ourjapanlife/findadoc-web
# 118 cleanup layout

Resolves #118


# What changed
Neaten up the app layout so it feels like we have something to build off of 
- delete old files
- simplify page names
- add an `app.vue` to create page structure
- update favicon
- add colors for tailwind
- use semantic header tags `h1` etc. 
- add a footer

NOTE: I am concerned tailwind isn't working correctly

# Testing instructions
This will require a visual inspection of the page layout, especially that the routing works as expected. Please confirm navigation between index and `about` and ensure that the correct content loads.
